.gitignore

```diff
@@ -116,3 +116,26 @@ packages/
 
 #JavaScript files
 *.jsbundle
+
+# Visual C++ cache files
+ipch/
+*.aps
+*.ncb
+*.opendb
+*.opensdf
+*.sdf
+*.cachefile
+*.VC.db
+*.VC.VC.opendb
+
+# Build results
+[Dd]ebug/
```

Comment:
>[Dd]ebug [](start = 0, length = 8)

Just make sure we don't already have a older called Debug somewhere (I think we did in the ReactNative C# project)